nss-day-cohort-48/giffygram-looksmart
# Al branch testing

#### Changes Made
1. Modified file `provider.js` to send messages to API, as well as `MessageForm.js` to get messages saved as objects before they are shipped off
​
#### Steps to Review
1. Checkout this branch locally.
    ```
    git fetch --all
    git checkout al-branch-testing
    ```
2. Open a new Terminal tab (⌘T) and navigate to the server directory.
3. Test app functionality.
    > When you send a message, it should communicate with API
4. View code file.
    > Confirm file modifications are present as indicated above.
    > Confirm no unused code or extraneous comments exist.
